File: src/main/java/model/Artist.java
package model;

public class Artist extends Entity{
    private String name;

    public Artist(String name) {
        this.name = name;
    }

    public Artist(Long artistID, String name) {
        setID(artistID);
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: MusicFestivalMPP/model/Artist.cs
namespace MusicFestivalMPP.model;

public class Artist: Entity
{
    public String Name { get; set; }

    public Artist(long id, string name) : base(id)
    {
        Name = name;
    }

    public Artist(string name)
    {
        Name = name;
    }
}